### ISSUE TITLE
Questions about baselines

### ISSUE THREAD

**Issue by zhiyuanyou** [chencn2020/SEAGULL#4]
Thanks for your wonderful work! I like it very much!

However, I am a little confused about the baselines. 

- Q-Align is trained to regress the quality score. How do you use it to obtain the Distortion Severity Degree and Distortion Type Labels ?
- For Q-Align and Q-Instruct, how do you input the region information ?

Thanks for your time.

**Comment by chencn2020** [comment]
Thank you for your attention to our work! 😊

For question 1, we retrained Q-Align on the SEAGULL-3k dataset, as Q-Align is also based on LLMs. This makes it able to predict Distortion Severity Degree and Distortion Type Labels.

For question 2, we draw a **rectangular box** on the input image to specify the region. The prompt is similar to "Please analyze the quality of this region within the rectangle."  
BTW, we chose a rectangular box instead of a mask boundary because the latter might result in the loss of some edge details within the region.

If you have any further questions or need additional clarification, please don't hesitate to contact us!

**Comment by zhiyuanyou** [comment]
Thanks for your detailed responses! 😊

For the first question, in some of our own experiments, the influences of model architecture are very small in the performance, and the dataset is the key factor. So, the advantage of Seagull to mPLUG-Owl2 (Q-Align) is from: (1) pre-training on SEAGULL-100w, (2) mask-based feature extractor. Is my understanding right ?

For the second question, if the image is cropped, how can the model assess the importance score, since the model can not see the background.

**Comment by zhiyuanyou** [comment]
Please forgive me for another question. 

In Table 1, for baselines like mPLUG-Owl2 (7B), LLaVA-1.5 (7B), Qwen2-VL (7B), do you retrain them on the SEAGULL-3k dataset ? I remember that, in many previous work, their performance on IQA is not so good.

**Comment by chencn2020** [comment]
> Thanks for your detailed responses! 😊
> 
> For the first question, in some of our own experiments, the influences of model architecture are very small in the performance, and the dataset is the key factor. So, the advantage of Seagull to mPLUG-Owl2 (Q-Align) is from: (1) pre-training on SEAGULL-100w, (2) mask-based feature extractor. Is my understanding right ?
> 
> For the second question, if the image is cropped, how can the model assess the importance score, since the model can not see the background.

For the first question, I think the model architecture also plays an important role. Different from mPLUG-Owl2 (Q-Align), which takes an image and a text prompt as input, SEAGULL additionally takes a mask as input. This makes it more reasonable for indicating the ROIs.

For the second question, only the inputs for vision-based models (e.g., HyperIQA) are cropped, as these model architectures do not support the same input format as VLM-based models (i.e., a full image with a bounding box and a text prompt to indicate the quality assessment for a region). This makes vision-based models show limited performance on this task.

**Comment by chencn2020** [comment]
> Please forgive me for so frequent questions.
> 
> In Table 1, for baselines like mPLUG-Owl2 (7B), LLaVA-1.5 (7B), Qwen2-VL (7B), do you retrain them on the SEAGULL-3k dataset ? I remember that, in many previous work, their performance on IQA is not so good.

It doesn’t matter. I’m happy to answer your questions :)

All the baselines in Table 1 have been retrained on SEAGULL-3k.

Let me know if you need further clarification!

**Comment by zhiyuanyou** [comment]
Thanks for your responses!
